sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/LocationSample.md

```diff
@@ -0,0 +1,17 @@
+# Location verify
+
+```C# Snippet:APC_Sample_LocationVerifyTest
+string ApcGatewayId = "/subscriptions/abcdefgh/resourceGroups/dev-testing-eastus/providers/Microsoft.programmableconnectivity/gateways/apcg-eastus";
+Uri _endpoint = new Uri("https://your-endpoint-here.com");
```

Comment:
Consider shortening the long lines to avoid scroll bars when viewing on GitHub.    
I'd also suggest more white space for readability.  This sample is very dense.

Reply:
have reduced the samples to max 120 lines.